ex6/ex6.c

```diff
@@ -22,6 +22,21 @@ and `clock_gettime()` should work just fine.
 int main()
 {
     // Your code here
+
+    // finds the average time it takes for my computer to make a system call an empty write to stdout
+    // program should use 'clock_gettime' procedure to time how long a single system call takes
+    	// this method will execute for 1 million iterations and then find the average of all the iterations in nanoseconds
+
+	struct timespec requestStart, requestEnd;
+	clock_gettime(CLOCK_REALTIME, &requestStart);
+	// function_call();
+	sleep(1);
+	clock_gettime(CLOCK_REALTIME, &requestEnd);
+
+	double accum = ( requestEnd.tv_sec - requestStart.tv_sec)
```

Comment:
It looks like your program only times a single system call. To get a more accurate idea of how long a system call takes on your machine and to account for variance, you'll want to sample many calls and then take the average time of all those calls. This is why there's a `number_iters` declaration at the top, to specify the number of iterations you should perform, with each iteration performing a system call and then accumulating the time it took.